evalml/tests/automl_tests/test_automl.py

```diff
@@ -1264,15 +1268,15 @@ class Pipeline2(DummyPipeline):
     if objective.name.lower() == "cost benefit matrix":
         automl = AutoMLSearch(X_train=X, y_train=y, problem_type=problem_type_value, max_iterations=3,
                               allowed_pipelines=[Pipeline1, Pipeline2], objective=objective(0, 0, 0, 0),
-                              additional_objectives=[], n_jobs=1)
+                              additional_objectives=[], train_best_pipeline=False, n_jobs=1)
```

Comment:
Mocking fit for time series regression resulted in the following error when trying to fit the best pipeline during training: `AttributeError: 'NoneType' object has no attribute 'fit'`. The pipeline didn't have a `self._component_obj` value.